server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestDeleteSnapshotAction.java

```diff
@@ -45,6 +45,9 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
         String[] snapshots = Strings.splitStringByCommaToArray(request.param("snapshot"));
         DeleteSnapshotRequest deleteSnapshotRequest = new DeleteSnapshotRequest(repository, snapshots);
         deleteSnapshotRequest.masterNodeTimeout(getMasterNodeTimeout(request));
+        deleteSnapshotRequest.waitForCompletion(
+            request.paramAsBoolean("wait_for_completion", DeleteSnapshotRequest.DEFAULT_WAIT_FOR_COMPLETION)
```

Comment:
Rather than exposing the default as a constant like that we often do this:

```suggestion
            request.paramAsBoolean("wait_for_completion", deleteSnapshotRequest.waitForCompletion())
```